src/operation/abort_transaction/mod.rs

```diff
@@ -28,11 +30,12 @@ impl AbortTransaction {
 
 impl Operation for AbortTransaction {
     type O = ();
+    type Command = Document;
     type Response = CommandResponse<WriteConcernOnlyBody>;
 
     const NAME: &'static str = "abortTransaction";
 
-    fn build(&mut self, _description: &StreamDescription) -> Result<Command> {
+    fn build(&mut self, _description: &StreamDescription) -> Result<Command<Self::Command>> {
```

Comment:
I don't think the addition of `Self::Command` here is necessary because the default `T` for `Command` is already `Document` -- can we revert this change or change the rest of the operations whose `build` signature wasn't updated (e.g. `ListCollections`) for consistency?

Reply:
done.